fedn/fedn/utils/pytorchmodel.py

```diff
@@ -1,55 +1,51 @@
 import os
 import tempfile
 import torch
```

Comment:
torch is not needed right? If so we need to discuss, since we have removed ML-library dependencies in combiner and reducer.